mkosi.md

```diff
@@ -913,10 +913,12 @@ local directory:
   This script has network access and may be used to install packages
   from other sources than the distro's package manager (e.g. pip, npm, ...),
   after all software packages are installed but before the image is
-  cached (if incremental mode is enabled).
-  Note that this script is executed directly in the image context with
-  the final root directory in place, without any `$SRCDIR`/`$DESTDIR`
-  setup.
+  cached (if incremental mode is enabled). This script is executed within
+  `$SRCDIR`. Contrary to usual behavior, you should install packages
+  to the system (`sudo pip install`, `npm install -g`) instead of in `$SRCDIR`
```

Comment:
I think formulating this as a should is a bit strong, how about "In contrast to a general purpose installation, it is safe to install packages to the system (`sudo pip install`, `npm install -g`) instead of in `$SRCDIR`..."